nautobot/extras/tests/test_filters.py

```diff
@@ -967,6 +969,190 @@ def test_destination_id(self):
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
 
 
+class RelationshipModelFilterSetTestCase(TestCase):
```

Comment:
Why not inherit from `FilterTestCases` here? Curious because all of the other test cases in this module do!